### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by Oransj**

### CODE REVIEW

**Review by z-rayc — CHANGES_REQUESTED** [review]
- Horizontally center tooltip relative to button
- Fix responsive width of profile page
- Update style to look a bit more like the main quiz page (i.e. remove purple color, maybe add a bit more spacing between elements).
- Why is there terms of service on profile page? Remove

File: package-lock.json
Comment (Molnes) [inline package-lock.json:1]: remove changes to this

File: package.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "devDependencies": {
+    "autoprefixer": "^10.4.19",
+    "postcss": "^8.4.38",
+    "tailwindcss": "^3.4.3"
+  }
+}
```
Comment (Molnes) [inline package.json:7]: remove changes to these aswell

**Review by Molnes — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Oransj) [reply package.json]: done

Comment (Molnes) [reply package.json]: fixed it for you

**Review by z-rayc — APPROVED** [review]

**Pull request merged**